Speedykom/Regional-Pandemic-Analytics
# Fix/keycloak role not updated

This PR resolves the issue of the role updates done in #311 which were not visible on the Dev Server.

Issues:
- Keycloak volume on dev server contains persistant data that includes added users, permissions, roles etc, removing the volume results in data loss
- realm.json doesn't contain the config existing in the dev server, removing keycloak volumes and building the image will apply a new config 

Solution:
- Exporting the realm config on the dev server
- Replace realm.json with it 

Testing Instructions :
- Remove Keycloak volume ( from dev server )
- Build the project , check if all the data needed for dev server exists
